applications/GeoMechanicsApplication/custom_strategies/schemes/SchemeStructure.puml

```diff
@@ -33,22 +33,32 @@ class GeneralizedNewmarkScheme
 class NewmarkQuasistaticPwScheme
 {
     void UpdateVariablesDerivatives() override;
+    void CalculateSystemContributions() override;
+    void CalculateRHSContribution() override;
 }
 class GeneralizedNewmarkTScheme
 {
     void UpdateVariablesDerivatives() override;
+    void CalculateSystemContributions() override;
+    void CalculateRHSContribution() override;
 }
 class NewmarkDynamicUPwScheme
 {
     void UpdateVariablesDerivatives() override;
+    void CalculateSystemContributions() override;
+    void CalculateRHSContribution() override;
 }
 class NewmarkQuasistaticDampedUPwScheme
 {
     void UpdateVariablesDerivatives() override;
+    void CalculateSystemContributions() override;
+    void CalculateRHSContribution() override;
 }
 class NewmarkQuasistaticUPwScheme
 {
     void UpdateVariablesDerivatives() override;
+    void CalculateSystemContributions() override;
+    void CalculateRHSContribution() override;
```

Comment:
If I understand correctly, I think the extra functions `CalculateSystemContributions` and `CalculateRHSContribution` should only be added to the damped and dynamic scheme right (so not the quasistatic, Pw and T schemes)?

Reply:
indeed. Done